EIPS/eip-3668.md

```diff
@@ -3,7 +3,7 @@ eip: 3668
 title: "CCIP Read: Secure offchain data retrieval"
 author: Nick Johnson (@arachnid)
```

Comment:
Can't comment on the correct place, but this EIP seems to be out of sync with the latest template. One apparent difference is the presence of the "Simple Summary" section and the lack of the `description:` preamble.